apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet/Trace/CloudTrace.cs

```diff
@@ -121,7 +121,7 @@ private CloudTrace(string projectId, TraceConfiguration config = null, TraceServ
         /// Initialize tracing for this application.
         /// </summary>
         /// <param name="application">The Http application.</param>
-        /// <param name="projectId">Optional if running on Google App Engine or Google Compute Engine.
+        /// <param name="projectId">Optional if unspecified and running on Google App Engine or Google Compute Engine.
```

Comment:
"unspecified and " does not really make sense here: if it was unspecified, it must have been optional :)

It should be added on the line below, so it says "If **unspecified and** running on GAE or...". Same for others as well.